### DISCUSSION TITLE
Theming markup

### DISCUSSION THREAD

**Discussion by hyptx** [KlipperScreen/KlipperScreen#898] [General]
I just got done creating a custom theme and was wondering if here was a way to see the html output in order to better write new css rules for the interface. I'm new to pi stuff but am a web dev by trade.

**Comment by alfrix** [comment]
use GtkInspector and you edit the stylesheet in realtime, you can launch Ks in a desktop env. just add `width=` and ` height=` in [main] to have it windowed
